guides/vectorizing_guide.ipynb

```diff
@@ -0,0 +1,3643 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install -U RelevanceAI-dev
</pre><p>Users should not see this. Please remove this output. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RelevanceAI/RelevanceAI/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>